Make extractOriginal() consistent with getOriginal().

markstory opened the pull request:
Make the two methods consistent. Introduce a new method for getting the original value of properties that have been modified.

I'm not very happy with the new method name, and would like suggestions if people can think of a better/clearer name.

Refs #6467

markstory committed:
Make extractOriginal() consistent with getOriginal().

Make the two methods consistent. Introduce a new method for getting
the original value of properties that have been modified.

Refs #6467

antograssiot commented:
:+1:

markstory commented:
Phpcs failures are not related but I will fix them on master.

lorenzo commented:
I'm still not happy with the name either, will think of alternatives

jhli commented:
Initial? Untouched?

jhawkwind commented:
Would `getChangedOriginal()` and `extractChangedOriginal()` work as better names?

lorenzo commented:
I think `untouched` sounds good, what do you think @markstory ?

markstory commented:
@lorenzo I like that. I'll update this tonight.

markstory committed:
Rename method.

I think extractOriginalChanged() makes more sense than OriginalDirty
did. I considered calling the method extractUntouched() at Lorenzo's
suggestion, but the method gets properties that have been touched.

Refs #6467

markstory commented:
Updated. I went with `extractOriginalChanged()`. To me `untouched` would imply the properties have not been changed, while this method gets original values for properties that _have_ changed.

lorenzo commented on src/Datasource/EntityTrait.php:
```diff
@@ -526,7 +526,10 @@ public function extract(array $properties, $onlyDirty = false)
-     * stored in this entity, indexed by property name
+     * stored in this entity, indexed by property name.
+     *
+     * Properties that have are unchanged changed will be included in the
```
that have are unchanged?

markstory replied:
I'm good at english and stuff.. :blush:

markstory committed:
Fix wording.

antograssiot commented on src/Datasource/EntityTrait.php:
```diff
@@ -526,7 +526,10 @@ public function extract(array $properties, $onlyDirty = false)
-     * stored in this entity, indexed by property name
+     * stored in this entity, indexed by property name.
+     *
+     * Properties that are unchanged changed will be included in the
```
`unchanged changed` @markstory I guess you're tired :wine_glass:  :wink:
